app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  add_flash_types :success, :info, :warning, :danger
```

Comment:
Bootstrap supports four message types versus the standard three.  I'm trying to standardize everything to Bootstrap to maintain consistency.

![bootstrap messages](https://cloud.githubusercontent.com/assets/10736777/26645965/c022b86a-45ee-11e7-97ef-a8e347480e7a.png)